### ISSUE TITLE
Make the connection between a single task and the system-to-deploy more visible

### ISSUE THREAD

**Issue by Karinon** [FREVA-CLINT/freva-deployment#60]
I have to deal with quite some error messages when I try to deploy on a completely blank machine.

It would be helpful, if the single tasks of the deployment would actually mention to which system they belong.

E.g. I had an error message during:

```
TASK [Installing additional python packages]
```

At no point it is clear to me, that this task belongs to the deployment of the `Core` and it would be really helpful to see this connection, e.g.

```
TASK [CORE] [Installing additional python packages]
```

**Comment by antarcticrainforest** [comment]
Actually I have that information:

```
PLAY [vault] ****************************************************************************************************************************************************************

TASK [Gathering Facts] ******************************************************************************************************************************************************
ok: [regiklim-dev.dkrz.de]

TASK [Copying docker/podman wrapper script] *********************************************************************************************************************************
changed: [regiklim-dev.dkrz.de]

TASK [Registering systemctl path] *******************************************************************************************************************************************
ok: [regiklim-dev.dkrz.de]
```

Ansible tells you with `PALY [vault]` that everything that belongs to "vault" will is now being executed. If you think that's not enough we can try the adjust the playbook taks names.

**Comment by Karinon** [comment]
Ok, probably I was just blind there. But I wouldn't mind to have the `PLAY`-Name in the `TASKS` aswell

**Comment by antarcticrainforest** [comment]
Should I do something about it or is this solved now?

**Comment by antarcticrainforest** [comment]
Addressed in #105
